src/components/ResourceList/components/FilterControl/tests/FilterControl.test.tsx

```diff
@@ -637,6 +637,37 @@ describe('<FilterControl />', () => {
     });
   });
 
+  describe('placeholder', () => {
+    it('renders default text if no placeholder passed.', () => {
+      const filterControl = mountWithAppProvider(
+        <Provider value={mockDefaultContext}>
+          <FilterControl {...mockDefaultProps} filters={mockFilters} />
+        </Provider>,
+      );
+
+      expect(filterControl.find(TextField).prop('placeholder')).toBe(
+        'Search items',
```

Comment:
You'll need to translate that string, but if we opt to not translate it and let the consumer pass the translate text then we just need to check that it was passed as is.

Reply:
I think if we go with the consumer doing the translation, then I am cool just testing the value passed as is.